draft/draft-ietf-bfd-yang.xml

```diff
@@ -852,46 +851,62 @@ INSERT_TEXT_FROM_FILE(../src/yang/example-mpls.xml)
 
       <t>/routing/control-plane-protocols/control-plane-protocol/bfd/ip-sh/sessions:
       data nodes local-multiplier, desired-min-tx-interval,
-      required-min-rx-interval, min-interval and authentication all impact the
-      BFD IP single-hop session.</t>
+      required-min-rx-interval and min-interval all impact the
+      BFD IP single-hop session. The source-addr and dest-addr data nodes can be used to
+      send BFD packets to unwitting recipients, <xref target="RFC5880"/> describes how
+      BFD mitigates against such threats. Authentication data nodes key-chain and meticulous
+      impact the security of the BFD IP single-hop session.</t>
 
       <t>/routing/control-plane-protocols/control-plane-protocol/bfd/ip-mh/session-group:
       the list specifies the IP multi-hop BFD session groups.</t>
 
       <t>/routing/control-plane-protocols/control-plane-protocol/bfd/ip-mh/session-group:
       data nodes local-multiplier, desired-min-tx-interval,
-      required-min-rx-interval, min-interval and authentication all impact the
-      BFD IP multi-hop session.</t>
+      required-min-rx-interval and min-interval all impact the
+      BFD IP multi-hop session. The source-addr and dest-addr data nodes can be used to
```

Comment:
Mahesh, thanks for going through the changes. I agree that this information (src/dest address) can be obtained via different means. The goal of this section is to identify what information is sensitive, not to discuss how it can be used or whether it can be obtained by other means or how to prevent an attack. I can see that one could view this information as "obvious" but IP addresses are sensitive information. So I'd rather add this information than debate with the IESG....

Reply:
Hi Reshad, those comments were made early on before the debate between Jeff, Ben and you started. I am with you, in that I do not want to let these comments hold up the draft. That is why I went ahead and approved the changes. 

Thanks for taking care of the comments.

Reply:
Thanks Mahesh.